_pages/index.md

```diff
@@ -3,16 +3,38 @@
 
 # About BIDS
 
-Neuroimaging experiments result in complicated data that can be arranged in many different ways. So far there is no consensus how to organize and share data obtained in neuroimaging experiments. Even two researchers working in the same lab can opt to arrange their data in a different way. Lack of consensus (or a standard) leads to misunderstandings and time wasted on rearranging data or rewriting scripts expecting certain structure. Here we describe a simple and easy to adopt way of organizing neuroimaging and behavioral data.
+Neuroimaging experiments result in complicated data that can be arranged in many different ways.
+So far there is no consensus how to organize and share data obtained in neuroimaging experiments.
+Even two researchers working in the same lab can opt to arrange their data in a different way.
+Lack of consensus (or a standard) leads to misunderstandings and time wasted on rearranging data or rewriting scripts expecting certain structure.
+With the Brain  Imaging Data Structure (BIDS), we describe a simple and easy to adopt way of organizing neuroimaging and behavioral data.
 
-![](./assets/img/dicom-reorganization-transparent-white_1000x477.png){: .center-image }
+![BIDS-folder-organization](./assets/img/dicom-reorganization-transparent-white_1000x477.png){: .center-image }
 
-BIDS is heavily inspired by the format used internally by the OpenfMRI repository (now known as [OpenNeuro.org](http://openneuro.org)). While working on BIDS we consulted many neuroscientists to make sure it covers most common experiments, but at the same time is intuitive and easy to adopt. The specification is intentionally based on simple file formats and folder structures to reflect current lab practices and make it accessible to a wide range of scientists coming from different backgrounds.
+BIDS is heavily inspired by the format used internally by the OpenfMRI repository (now known as [OpenNeuro.org](http://openneuro.org)).
```

Comment:
I propose either this

BIDS **is** heavily inspired by the format used internally by the [OpenNeuro.org](http://openneuro.org) repository (formerly known as OpenFMRI).

or

BIDS **was** heavily inspired by the format used internally by the OpenfMRI repository that is now known as [OpenNeuro.org](http://openneuro.org).